packages/core/src/Banner.js

```diff
@@ -74,7 +74,7 @@ const Banner = props => {
         )}
         <Box width={1}>
           <Text textAlign={props.textAlign}>
-            <Heading.h5 textStyle="display2">{props.header}</Heading.h5>
+            <Text textStyle="display2">{props.header}</Text>
```

Comment:
Why is this `Text` component inside another text component? Couldn't we just achieve the same by doing something like this? `<Text textAlign={props.textAlign} textStyle="display2">{props.header}</Text>`